tests/conftest.py

```diff
@@ -83,6 +83,24 @@ def fixture(request):
     generate_fixture_group(name, variants)
 del generate_fixture_group
 
+# ------------------------------------------------------------------------------
+#                Adding a customizable output dir for plots
+# ------------------------------------------------------------------------------
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        "--systemtest_outdir", action="store", default="./test_report/generated/plots"
+    )
```

Comment:
Very nice! Let's broaden the scope of this and call that fixture `artefacts_dir`: this way, it's clear that all artefacts are to be exported to this directory, not only system test-related plots.

I suggest to set the default to `build/tests/artefacts`. Side question: what happens with this default if you call pytest in another directory that `$ERADIATE_DIR`?

Reply:
I updated this part. The default is now $ERADIATE_DIR as long as that is set and otherwise defaults to the current working directory. But to be honest, if you try to run the tests without the eradiate conda environment, which sets that, the tests won't run anyways.

Reply:
Sounds good to me.